### ISSUE TITLE
undefined method in ddd.py?

### ISSUE THREAD

**Issue by Tibazz** [pytrip/pytrip#324]
https://github.com/pytrip/pytrip/blob/master/pytrip/ddd.py#L36
this method is not defined for this class it seems?

**Comment by grzanka** [comment]
Yes, it is not defined and this is a bug ! Thanks for pointing that !

DDD class is not tested and not used by any other parts of the code, a bit abandoned :(

Do you need it ? If yes, please describe your case, we can fix it or provide some other solution.

**Comment by Tibazz** [comment]
I am working on a master-theisis where i need to use .ddd files inside a custom raycasting algorithm and interpolate them as i go along to achieve, amongst other things, dose distributions and i was looking into PyTrip98 to do those interpolations. So far i have made in-house code for quite a lot of things that frankly already exist inside PyTrip98 mostly because i cant figure out how to install it :) and also because i want to learn stuff by myself. These bugs i've found from copy-pasting the source code.

**Comment by grzanka** [comment]
Great ! We are happy to see new user of Pytrip. 
If you are patient enough to complain using issues on github we can help you to fix most of the missing parts in pytrip and improve the installation procedure.

On the other help you have some inhouse code that could enrich pytrip repo, please let us know (via issue), so we could discuss.

I will fix the problematic in next days. If you could paste here some use-case for DDD class I would check it and improve the code a bit.

**Comment by grzanka** [comment]
See PR #339
